### PR TITLE
Updated examples

### PR DESCRIPTION
**Pull request by milo157**

### CODE REVIEW

File: examples/stable_diffusion.mdx [hunk 1]
```diff
@@ -81,7 +92,41 @@ def predict(item, run_id, logger):
...
+
+```bash
+curl --location --request POST 'https://run.cerebrium.ai/v1/p-xxxxx/project/predict' \
+--header 'Authorization: dev-c_api_key-ad0064b953c8053ff7bc' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "url": "https://www.youtube.com/watch?v=UF8uR6Z6KLc&ab_channel=Stanford",
+    "question": "How old was Steve Jobs when started Apple?"
+}'
+```
```
Comment (elijah-rou) [inline examples/stable_diffusion.mdx:119]: this doesn't seem like it makes sense as a prompt for Stable Diffusion

**Review by elijah-rou — APPROVED** [review]
just the one thing

File: examples/segment_anything.mdx [hunk 2]
```diff
@@ -159,11 +165,32 @@ In the above code we do a few things:
 3. We run our "Segment Anything" model with the image the user sent through the request.
 4. Lastly, we responded with the converted images. All returns from your endpoint function must be of type list or JSON.
 
-We can then deploy our model with the following line of code
+We can then deploy our model to an A10 instance with the following line of code
 
 ```
-cerebrium-deploy -n stable-diffusion -h A10 -m 16 -k <API_KEY>
+cerebrium-deploy -n stable-diffusion -h A10 -k <API_KEY>
```
Comment (jonoirwinrsa) [inline examples/segment_anything.mdx:171]: @milo157 wrong name for this example right?

File: examples/segment_anything.mdx [hunk 3]
```diff
@@ -159,11 +165,32 @@ In the above code we do a few things:
...
 ```
 
 After a few minutes, your model should be deployed and an endpoint should be returned. Let us create a CURL request to see the response
 
+```
+curl --location --request POST 'https://run.cerebrium.ai/v1/p-xxxxxx/segment-anything/predict' \
+--header 'Authorization: dev-c_api_key-XXXXXXXXXXXX' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "file_url": "https://cdn-www.thefashionspot.com/assets/uploads/gallery/the-off-duty-male-models-of-milan-mens-fashion-week/milano-m-moc-rf14-8533.jpg",
```
Comment (jonoirwinrsa) [inline examples/segment_anything.mdx:181]: the-off-duty-male-models-of-milan-mens-fashion-week

File: examples/stable_diffusion.mdx [hunk 4]
```diff
@@ -81,7 +92,41 @@ def predict(item, run_id, logger):
...
+To deploy this function to a A10 instance we run the following code:
+
+```bash
+cerebrium-deploy -n stable-diffusion -h A10 -k <API_KEY>
+```
+
+We can then make the following request:
+
+```bash
+curl --location --request POST 'https://run.cerebrium.ai/v1/p-xxxxx/project/predict' \
```
Comment (jonoirwinrsa) [inline examples/stable_diffusion.mdx:112]: instead of "project" could use "stable-diffusion"

**Pull request merged**